GVRf/Sample/gvr-cubemap/src/org/gearvrf/cubemap/CubemapScript.java

```diff
@@ -60,57 +58,69 @@ public void onInit(GVRContext gvrContext) {
+        scene.addSceneObject(mCube);
+
+//        ///////////////////////////////////////////////////////////////
+//        // create surrounding cube using six GVRSceneOjbects (quads) //
+//        ///////////////////////////////////////////////////////////////
+//        FutureWrapper<GVRMesh> futureQuadMesh = new FutureWrapper<GVRMesh>(
+//                gvrContext.createQuad(CUBE_WIDTH, CUBE_WIDTH));
+//        
+//        GVRSceneObject mFrontFace = new GVRSceneObject(gvrContext,
+//                futureQuadMesh, futureCubemapTexture);
+//        mFrontFace.getRenderData().setMaterial(cubemapMaterial);
+//        mFrontFace.setName("front");
+//        scene.addSceneObject(mFrontFace);
+//        mFrontFace.getTransform().setPosition(0.0f, 0.0f, -CUBE_WIDTH * 0.5f);
+//
+//        GVRSceneObject backFace = new GVRSceneObject(gvrContext,
+//                futureQuadMesh, futureCubemapTexture);
+//        backFace.getRenderData().setMaterial(cubemapMaterial);
+//        backFace.setName("back");
+//        scene.addSceneObject(backFace);
+//        backFace.getTransform().setPosition(0.0f, 0.0f, CUBE_WIDTH * 0.5f);
+//        backFace.getTransform().rotateByAxis(180.0f, 0.0f, 1.0f, 0.0f);
+//
+//        GVRSceneObject leftFace = new GVRSceneObject(gvrContext,
+//                futureQuadMesh, futureCubemapTexture);
+//        leftFace.getRenderData().setMaterial(cubemapMaterial);
+//        leftFace.setName("left");
+//        scene.addSceneObject(leftFace);
+//        leftFace.getTransform().setPosition(-CUBE_WIDTH * 0.5f, 0.0f, 0.0f);
+//        leftFace.getTransform().rotateByAxis(90.0f, 0.0f, 1.0f, 0.0f);
+//
+//        GVRSceneObject rightFace = new GVRSceneObject(gvrContext,
+//                futureQuadMesh, futureCubemapTexture);
+//        rightFace.getRenderData().setMaterial(cubemapMaterial);
+//        rightFace.setName("right");
+//        scene.addSceneObject(rightFace);
+//        rightFace.getTransform().setPosition(CUBE_WIDTH * 0.5f, 0.0f, 0.0f);
+//        rightFace.getTransform().rotateByAxis(-90.0f, 0.0f, 1.0f, 0.0f);
+//
+//        GVRSceneObject topFace = new GVRSceneObject(gvrContext, futureQuadMesh,
+//                futureCubemapTexture);
+//        topFace.getRenderData().setMaterial(cubemapMaterial);
+//        topFace.setName("top");
+//        scene.addSceneObject(topFace);
+//        topFace.getTransform().setPosition(0.0f, CUBE_WIDTH * 0.5f, 0.0f);
+//        topFace.getTransform().rotateByAxis(90.0f, 1.0f, 0.0f, 0.0f);
+//
+//        GVRSceneObject bottomFace = new GVRSceneObject(gvrContext,
+//                futureQuadMesh, futureCubemapTexture);
+//        bottomFace.getRenderData().setMaterial(cubemapMaterial);
+//        bottomFace.setName("bottom");
+//        scene.addSceneObject(bottomFace);
+//        bottomFace.getTransform().setPosition(0.0f, -CUBE_WIDTH * 0.5f, 0.0f);
+//        bottomFace.getTransform().rotateByAxis(-90.0f, 1.0f, 0.0f, 0.0f);
 
```

Comment:
Those codes demonstrate a different way to construct a cube (i.e. using six quads instead of a cube object). It should be good to leave them here, so that the user can easily switch between the two ways.